app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_values = {}
+
+    def inner(*args) -> callable:
+        if args in cache_values:
+            print("Getting from cache")
+            return cache_values[args]
+        else:
+            cache_values[args] = func(*args)
```

Comment:
Your approach is incorrect: you should store arguments and the result of the function with these arguments.